### PR TITLE
Add support for Aqara FP1E.

### PR DESCRIPTION
**Pull request by morgendagen**
Fixes https://github.com/Koenkk/zigbee2mqtt/issues/23577

# Done
- [x] Basic support
- [x] Add "Restart device"
- [x] Dependency on https://github.com/Koenkk/zigbee-herdsman-converters/issues/8042

### CODE REVIEW

File: src/devices/lumi.ts [hunk 1]
```diff
@@ -1891,6 +1891,34 @@ const definitions: DefinitionWithExtend[] = [
         },
         extend: [lumiZigbeeOTA()],
     },
+    {
+        zigbeeModel: ['lumi.sensor_occupy.agl1'],
+        model: 'lumi.sensor_occupy.agl1',
```
Comment (Koenkk) [inline src/devices/lumi.ts:1896]:
```suggestion
        model: 'FP1E',
```

File: src/devices/lumi.ts [hunk 2]
```diff
@@ -1891,6 +1891,34 @@ const definitions: DefinitionWithExtend[] = [
...
+        vendor: 'Aqara',
+        description: 'Presence sensor FP1E',
```
Comment (Koenkk) [inline src/devices/lumi.ts:1898]:
```suggestion
        description: 'Presence sensor',
```

**Comment by Koenkk** [comment]
Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `FP1E.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by gredondogalan** [comment]
Hi, regarding this thread I have doubt checking the compatible devices in the Zwave2MQTT page.
I have not found available the FP1E model of aqara on this [page](https://www.zigbee2mqtt.io/supported-devices/#s=fp1), only the FP1 appears to me.
Does this mean that it is available to integrate the FP1E or not yet?

**Comment by morgendagen** [comment]
> Does this mean that it is available to integrate the FP1E or not yet?

I believe you have to be patient and await the next release of z2m.

**Pull request merged**